easybuild/easyconfigs/r/RELION/RELION-5.0.0_beta.2023.12.21-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,216 @@
+# Author: Jasper Grimm (UoY)
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '5.0.0_beta.2023.12.21'
+_commit = '90d239e'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page'
+description = """RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer
+ program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class
+ averages in electron cryo-microscopy (cryo-EM).
+ """
+
+build_info_msg = """To install RELION, it's recommended to change the defaults in this easyconfig,
+ to something suitable for your site. Recommended changes:
+   * Set '_torch_home' to the location where Blush model weights are downloaded (~10GB).
+     Instructions for downloading these can be found here: 
+     https://github.com/3dem/relion/issues/1003#issuecomment-1786280151
+
+   * Uncomment the following optional dependencies if desired:
+      - ctffind or GCTF dependency
+      - MotionCor2
+      - ResMap
```

Comment:
aren't gnuplot, Ghostscript, tcsh (for relion shell) also optional deps?

Reply:
for `ctffind` and `tcsh`, you might have to set `modextravars`
```
modextravars = { 
    # Common settings
    'RELION_CTFFIND_EXECUTABLE': 'ctffind',
    'RELION_SHELL': 'csh',              # Shell used to launch CTFFIND/GCTF in CtfFind jobs
```

Reply:
`gnuplot` and `tcsh` aren't mentioned in the 5.0 install docs, so I'm not sure?

I'd consider setting `'RELION_CTFFIND_EXECUTABLE': 'ctffind',` in the ctffind easyconfig

Reply:
for `tcsh`, I guess `bash` will probably work too, but csh is default, see https://relion.readthedocs.io/en/release-5.0/Installation.html#edit-the-environment-set-up
> RELION_SHELL: A shell used to launch CTFFIND/GCTF in CtfFind jobs ('csh' by default; only available from 3.1)

for `gnuplot`, I assume it's optional given https://relion.readthedocs.io/en/release-5.0/SPA_tutorial/Class3D.html#analysing-the-results-in-more-detail
> Alternatively, if gnuplot is installed on your system, you may type: ...

but those are minor things, don't let it delay this PR, nice work!

Reply:
default seems to be /bin/sh:
https://github.com/3dem/relion/blob/6a43bbb68f95a666a0769167b4342b2ee70e8692/src/ctffind_runner.cpp#L104

We never set RELION_SHELL at EMBL in the last few years and ctffind properly runs.

Reply:
I've removed the `RELION_SHELL` modextravar again